src/constants/jsonRpcEndpoints.ts

```diff
@@ -3,7 +3,20 @@ import { SupportedChainId } from './chains'
 /**
  * Fallback JSON RPC endpoints if integrator does not provide one
  */
-export const JSON_RPC_FALLBACK_ENDPOINTS: { [key in SupportedChainId]?: string[] } = {
+export const JSON_RPC_FALLBACK_ENDPOINTS: { [chainId: number]: string[] } = {
+  // KOVAN = 42,
+  // OPTIMISTIC_KOVAN = 69,
   [SupportedChainId.MAINNET]: ['https://cloudflare-eth.com'],
-  [SupportedChainId.RINKEBY]: ['https://cloudflare-eth.com/v1/rinkeby'],
+  [SupportedChainId.ROPSTEN]: ['https://rpc.ankr.com/eth_ropsten'],
+  [SupportedChainId.RINKEBY]: ['https://rpc.ankr.com/eth_rinkeby'],
+  [SupportedChainId.GOERLI]: ['https://rpc.ankr.com/eth_goerli'],
+  [SupportedChainId.ARBITRUM_ONE]: ['https://arbitrum.public-rpc.com'],
+  [SupportedChainId.ARBITRUM_RINKEBY]: ['https://rinkeby.arbitrum.io/rpc'],
+  [SupportedChainId.OPTIMISM]: ['https://rpc.ankr.com/optimism'],
+  [SupportedChainId.POLYGON]: [
+    'https://polygon-rpc.com',
+    'https://rpc-mainnet.matic.network',
+    'https://rpc-mainnet.matic.quiknode.pro',
+  ],
+  // [SupportedChainId.CELO]: ['https://rpc.ankr.com/celo'],
```

Comment:
separate PR: need to add CELO as supported chain to widget (mirror changes made in this interface PR https://github.com/Uniswap/interface/pull/3915)